test/external/external_test_opt.py

```diff
@@ -64,14 +64,14 @@ def test_enet_se(self):
     for p in get_parameters(model): p.assign(np.zeros(p.shape, dtype=p.dtype.np))
     img = Tensor.randn(1, 3, 224, 224)
     # TODO: this seems very high
-    with CLCache(115):
+    with CLCache(116):
       model.forward(img).realize()
 
   def test_resnet(self):
     model = ResNet18()
     for p in get_parameters(model): p.assign(np.zeros(p.shape, dtype=p.dtype.np))
     img = Tensor.randn(1, 3, 224, 224)
-    with CLCache(26):
+    with CLCache(29):
```

Comment:
Hmm, that's a lot for a forward pass. Is it faster?

Reply:
so i reverted the default change and replaced with a env var. I tested around and some kernels became slower (gemm 4096x4096 became 3X slower), so i don't see a universal good default. benchmarking here is tricky because it might be faster with handcode but slower with best beam.